src/session/sidebar/mod.rs

```diff
@@ -54,6 +54,7 @@ mod imp {
         type ParentType = gtk::Widget;
 
         fn class_init(klass: &mut Self::Class) {
+            User::static_type();
```

Comment:
These are distributed around the whole application, what makes it harder to keep track. Would it make sense to initialize these in one single place?